driver/controller.go

```diff
@@ -191,6 +197,19 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	if snapshot != nil && volumeReq.SizeGigaBytes > int64(snapshot.SizeGigaBytes) && volumeReq.SizeGigaBytes > 1 {
+		_, _, err := d.storageActions.Resize(ctx, vol.ID, int(volumeReq.SizeGigaBytes), volumeReq.Region)
+		log.Info("resizing volume because its requested size is larger than the size of the backing snapshot")
+		if err != nil {
+			return nil, status.Errorf(codes.Internal, "cannot resize volume %s: %s", vol.ID, err.Error())
+		}
```

Comment:
Where did the action handling go?

Reply:
I removed it because it was hanging indefinitely and returning the following error. 
```text
May  8 15:18:53 |ERRO| method failed method=/csi.v1.Controller/CreateVolume error=rpc error: code = Internal desc = failed waiting on action ID 1782594914 for volume ID a056bdc3-edb3-11ed-867f-0a58ac14d19d to get resized: timed out waiting for the condition 
```

Reply:
We do this elsewhere though: https://github.com/digitalocean/csi-digitalocean/pull/490#discussion_r1187661458

If an action is returned, then I'd strongly believe it's always supposed to be used. If that fails consistently, then we should try to understand why that's the case.